Lib/json/tool.py

```diff
@@ -40,7 +40,7 @@ def main():
                 obj = json.load(infile,
                                 object_pairs_hook=collections.OrderedDict)
         except ValueError as e:
-            raise SystemExit(e)
+            raise SystemExit(e) from None
```

Comment:
Agree. It just doesn't have any visible effect. The original exception is passes to SystemExit, and `str(e)` is printed to stderr at exit. `__cause__` and `__context__` are ignored.